src/sidebar/store/modules/test/drafts-test.js

```diff
@@ -0,0 +1,200 @@
+'use strict';
+
+const immutable = require('seamless-immutable');
+
+const drafts = require('../drafts');
+const { Draft } = require('../drafts');
+const createStore = require('../../create-store');
+
+const fixtures = {
+  draftWithText: immutable({
```

Comment:
I think this code was just copied from the previous tests, but I'll note that I think it would have been possible to wrap the entire fixtures object with `immutable` (ie. `fixtures = immutable({ ... })`)

Reply:
roger